app/controllers/community_controller.rb

```diff
@@ -9,12 +9,12 @@ def index
       mentoring:  'Helping as a Mentor',
       organizing: 'Helping as an Organizer'
     }
+
+    @countries = User.pluck(:country).uniq
     @users = User.ordered(params[:sort], params[:direction])
```

Comment:
I think it would be a bit more readable if only those things later to be used in the view would be instantiated as instance variables. So how about having:
```ruby
users = User.order...
communtiy = CommunitUsers.new(..., users)
@users = community.all
```

What do you think?